mantle/kola/tests/podman/podman.go

```diff
@@ -216,12 +216,11 @@ func podmanWorkflow(c cluster.TestCluster) {
 	// Test: Delete container
 	c.Run("delete", func(c cluster.TestCluster) {
 		cmd := fmt.Sprintf("sudo podman rmi %s", image)
-		out := c.MustSSH(m, cmd)
-		imageID := string(out)
+		c.MustSSH(m, cmd)
 
-		cmd = fmt.Sprintf("sudo podman images | grep %s", imageID)
+		cmd = fmt.Sprintf("sudo podman image exists %s", image)
 		out, err := c.SSH(m, cmd)
-		if err == nil {
+		if err != nil {
```

Comment:
image exists will return success if it found the image, which in this case would be an error.

```suggestion
		if err == nil {
```